pkg/function/client/client.go

```diff
@@ -73,8 +71,8 @@ func New(inputOptions ...Option) (*client, error) {
 			grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(opts.maxMessageSize), grpc.MaxCallSendMsgSize(opts.maxMessageSize)),
 		)
 	} else {
-		log.Println("UDS Client ", function.UDS, opts.sockAddr)
-		sockAddr = fmt.Sprintf("%s:%s", function.UDS, opts.sockAddr)
+		log.Println("UDS Client ", function.UDS, opts.udsSockAddr)
```

Comment:
```suggestion
		log.Println("UDS Client:", function.UDS, opts.udsSockAddr)
```
does it make sense to print `socketAddr` in the log?.. if yes please fix for multprocessing tcp client too

Reply:
Done